nuxeo-mongodb-audit/src/main/java/org/nuxeo/mongodb/audit/pageprovider/MongoDBDocumentHistoryPageProvider.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * (C) Copyright 2017 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Kevin Leturc
+ */
+package org.nuxeo.mongodb.audit.pageprovider;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.SortInfo;
+import org.nuxeo.ecm.platform.audit.api.document.AdditionalDocumentAuditParams;
+import org.nuxeo.ecm.platform.audit.api.document.DocumentAuditHelper;
+
+/**
+ * @since 9.1
+ */
+public class MongoDBDocumentHistoryPageProvider extends MongoDBAuditPageProvider {
+
+    private static final long serialVersionUID = 1L;
+
+    private static final Log log = LogFactory.getLog(MongoDBDocumentHistoryPageProvider.class);
+
+    public static final String SINGLE_QUERY = "{ \"docUUID\": \"?\" }";
+
+    public static final String COMPLEX_QUERY = "{ \"$or\": [ { \"docUUID\": \"?\" }, { \"$and\": [ { \"docUUID\": \"?\" }, { \"eventDate\": { \"$lte\": ISODate(\"?\") } } ] } ]}";
```

Comment:
Would you mind using the constants like `PROPERTY_DOC_UUID` inside these strings too?